stacker_blueprints/dynamodb.py

```diff
@@ -0,0 +1,152 @@
+from stacker.blueprints.base import Blueprint
```

Comment:
Woah, Infrastructure wasn't listed as a collaberator here.  You should be good to edit now, @jeffpierce